travis/install-common-deps.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
```

Comment:
Copied from the old .travis.yml `install` section ([see comment](https://github.com/mlflow/mlflow/pull/1097/files#r273299584))